### PR TITLE
UPdated options for WSL integration with docker

### PR DESCRIPTION
**Pull request by johnthompson365**

### CODE REVIEW

**Comment by hanzei** [comment]
@svelle Can you review this? I'm not sure who else uses windows.

File: site/content/contribute/server/developer-setup/windows-wsl.md [hunk 1]
```diff
@@ -5,8 +5,10 @@ This is an unofficial guide. Community testing, feedback and improvements are we
 2. Install and setup Docker:
 
     * Install Docker for Windows: https://docs.docker.com/docker-for-windows/
-    * Link Windows Subsystem for Linux to Docker for Windows: https://medium.com/@sebagomez/installing-the-docker-client-on-ubuntus-windows-subsystem-for-linux-612b392a44c4.
-        * You should end up with the Docker client running on Linux (WSL) sending commands to your Docker Engine daemon installed on Windows.
+    * Link Windows Subsystem for Linux to Docker for Windows: 
+            TCP option: https://medium.com/@sebagomez/installing-the-docker-client-on-ubuntus-windows-subsystem-for-linux-612b392a44c4.
+            More secure option: https://devblogs.microsoft.com/commandline/cross-post-wsl-interoperability-with-docker/
+        * With wither option you should end up with the Docker client running on Linux (WSL) sending commands to your Docker Engine daemon installed on Windows.
```
Comment (svelle) [inline site/content/contribute/server/developer-setup/windows-wsl.md]:
```suggestion
        * With either option you should end up with the Docker client running on Linux (WSL) sending commands to your Docker Engine daemon installed on Windows.
```
Small typo, i think?

Comment (johnthompson365) [reply site/content/contribute/server/developer-setup/windows-wsl.md]: Correct.

Comment (jasonblais) [reply site/content/contribute/server/developer-setup/windows-wsl.md]: @johnthompson365 when you have a chance, can you help push the commit suggestion to your branch? I tried but didn't have permissions to. Thanks for the documentation!

Comment (johnthompson365) [reply site/content/contribute/server/developer-setup/windows-wsl.md]: Will do.

**Comment by johnthompson365** [comment]
@cpanato can this be merged based on previous review?

**Comment by cpanato** [comment]
@johnthompson365 asked the reviewers to look again

File: site/content/contribute/server/developer-setup/windows-wsl.md (hunk 1 above)
Comment (jasonblais) [reply site/content/contribute/server/developer-setup/windows-wsl.md]: I still see the typo

**Comment by johnthompson365** [comment]
Updated the typo!

**Pull request closed without merging**